packages/maskbook/src/utils/createNormalReactRoot.tsx

```diff
@@ -0,0 +1,20 @@
+import ReactDOM from 'react-dom'
+import { ErrorBoundary } from '../components/shared/ErrorBoundary'
+import { NoEffectUsePortalShadowRootContext } from '@masknet/theme'
+
+export function createNormalReactRoot(jsx: JSX.Element, container?: HTMLElement) {
+    if (!container) container = document.getElementById('root') ?? void 0
+    if (!container) {
+        container = document.createElement('div')
+        document.body.appendChild(container)
+    }
+    if (process.env.NODE_ENV === 'development') {
+        // Make the document cleaner
+        setTimeout(() => [...document.querySelectorAll('script')].forEach((x) => x.remove()), 200)
```

Comment:
Need clean the timer and refactor it friendly to `startEffects`.

Reply:
It's used to clean up the HTML in the dev tools because we will have many `<script>` tags due to webpack. I don't think we should clean up this.